hazelcast/src/test/java/com/hazelcast/scheduledexecutor/impl/ScheduledExecutorServiceBasicTest.java

```diff
@@ -606,6 +606,61 @@ public void capacity_whenAutoDisposable() throws Exception {
         // no exceptions thrown
     }
 
+    @Test
+    public void capacity_whenAutoDisposableRunnable() {
+        String schedulerName = ANY_EXECUTOR_NAME;
+
+        HazelcastInstance[] instances = createClusterWithCount(1, null);
+        IScheduledExecutorService service = instances[0].getScheduledExecutorService(schedulerName);
+        List<IScheduledFuture> futures = new ArrayList<>();
+        for (int i = 0; i < 10; i++) {
+            futures.add(service.schedule(autoDisposable(new PlainRunnableTask()), 0, TimeUnit.SECONDS));
+        }
+
+        futures.forEach(this::assertTaskHasBeenDestroyedEventually);
+
+        // Re-schedule to verify capacity
```

Comment:
Afaiu, since the scheduler capacity is not set to a small number in this test, this capacity verification doesn't work here (I think this could pass in the old version).

Reply:
I've removed this test because below is the same test but with the capacity set

Reply:
There were some nuance differences between these two tests, the below one was testing operations that are always scheduled on the same member (on the member owning the partition corresponding to the specified key), while the removed test's operations were scheduled on any random member. I think it would be better to adapt the names of the tests in this aspect and leave both after we also set the capacity in the first one.